### DISCUSSION TITLE
(SEO) addon or package for keyword density?

### DISCUSSION THREAD

**Discussion by jensolafkoch** [statamic/cms#5157] [Q&A]
We use SEO Pro and a client asked for an additional keyword density tool. 

It seems that no current Statamic SEO addon or any useful Laravel or PHP package is out there doing the job (but maybe I'm wrong).

If someone has any idea if there is something available that we can use? Any tipps appreciated!

(I found https://github.com/PiedWeb/TextAnalyzer so far, which might help to build something from scratch ...)

**Comment by edalzell** [comment]
I'd submit feature requests on the SEO addon repos.

**Reply by edalzell** [reply]
or reach out to a dev to build or enhance one of them.

**Comment by jensolafkoch** [comment]
Thanks, Erin! If the client really insists, we'd probably try to build it ourselves, depending on complexity. Feature request for SEO Pro would be a good alternative though.
